app/src/main/res/values-de/strings.xml

```diff
@@ -44,6 +44,14 @@
     <string name="dark">Dunkel</string>
     <string name="x_deleted">%1$s gelöscht</string>
     <string name="undo">Abbrechen</string>
+	<string name ="normal">Normal</string>
```

Comment:
Please remove this additional space here to align this line with the others.

Reply:
fixed this, but isn't it wrong that all new strings look like 

> <string name ="

i think the space between name and = is to much, so i've changed them now to 

> <string name="